src/deployment-guides/tap.md

```diff
@@ -453,110 +479,57 @@ export TAP_REGISTRY_SERVER=<registry server uri>
 export TAP_REGISTRY_USER=<registry_user>
 export TAP_REGISTRY_PASSWORD=<registry_password>
 export TAP_GITHUB_TOKEN=<github_token>
-export TAP_APP_DOMAIN=<app_domain>
+export tap_view_app_domain=<sub domain of view cluster>
```

Comment:
Same comment as above Please make this all-caps and adjust all references as such.

Reply:
done